lib/src/facts/resolvers/networking_resolver.cc

```diff
@@ -46,6 +49,16 @@ namespace facter { namespace facts { namespace resolvers {
 
     void networking_resolver::resolve(collection& facts)
     {
+       
+        boost::program_options::variables_map vm;
+        auto hocon_conf = load_default_config_file();
+        load_fact_settings(hocon_conf, vm);
+        string interface_filter = "";
```

Comment:
using default constructor `string interface_filter;` will initialise to empty string